src/instance/misc.js

```diff
@@ -39,6 +39,14 @@ exports._applyFilters = function (value, oldValue, filters, write) {
   return value
 }
 
+exports._bubbleChanges = function (value, expression) {
```

Comment:
As for expression check - a reverse write should be applied as long as the expression starts with the alias. e.g. `item.test` and `item[test]` both should trigger reverse write. So you need a regular expression, e.g. `new RegExp(alias + '\\b')`.